tests/integration/devfile/cmd_devfile_deploy_test.go

```diff
@@ -47,9 +48,12 @@ var _ = Describe("odo devfile deploy command tests", func() {
 			helper.CmdShouldPass("odo", "create", "nodejs", "--project", namespace, cmpName)
 			helper.CmdShouldPass("odo", "url", "create", "--port", "3000")
 
+			imageTag = fmt.Sprintf("image-registry.openshift-image-registry.svc:5000/%s/my-nodejs:1.0", namespace)
 			helper.CopyExampleDevFile(filepath.Join("source", "devfilesV2", "nodejs", "devfile.yaml"), filepath.Join(context, "devfile.yaml"))
-			output := helper.CmdShouldPass("odo", "deploy", "--tag", imageTag, "--devfile", "devfile.yaml")
+			output := helper.CmdShouldPass("odo", "deploy", "--tag", imageTag, "devfile.yaml", "-v", "4")
```

Comment:
Is "devfile.yaml" a valid argument that can be passed to odo deploy? Does it default to ./devfile.yaml already?